packages/dashboard/src/components/appbar.tsx

```diff
@@ -213,26 +208,21 @@ export const AppBar = React.memo(({ extraToolbarItems }: AppBarProps): React.Rea
       AppEvents.refreshAlert.subscribe({
```

Comment:
It should just subscribe to `alertListOpenedAlert` instead of `refreshAlert` (which should be removed).

Reply:
This subscription is to update the number on the badge to show how many unacknowledged alerts there are. The event is triggered when a new alert comes in, or when someone responses to one.

![image](https://github.com/open-rmf/rmf-web/assets/5383623/0d0f2201-4009-4412-8bfa-ba46768efaac)

Reply:
My bad, in that case, it should subscribe to alert requests and alert response and build the list from there.

Reply:
that will be more like dead-reckoning though, just counting up and down from the initial state of the dashboard. Having an API call to get the exact number is more straightforward imo